pkg/daemon/daemon.go

```diff
@@ -580,8 +590,17 @@ func (dn *Daemon) completeDrain() error {
 		return err
 	}
 
+	if utils.ClusterType == utils.ClusterTypeOpenshift {
+		glog.Infof("completeDrain(): resume MCP %s", dn.mcpName)
```

Comment:
Making sure I understand this properly, the completeDrain() function gets called after the reboot correct? To uncordon the node?

Reply:
Yes. It will be executed after reboot or after draining without a reboot.